modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15545,6 +15547,7 @@ search-file-entries=Search File Entries
 search-folder=Search Folder
 search-folders=Search Folders
 search-for=Search for
+search-for-a-role=Search for a role
```

Comment:
Hey, Dan and Larissa.

I know that some of these languages ​​will be used as placeholders, but can we check if we should use title case or sentence case for these cases?

Reply:
we decided to treat them as sentences, thanks Igor!